src/mame/drivers/agat.cpp

```diff
@@ -0,0 +1,1107 @@
+// license:BSD-3-Clause
+// copyright-holders:R. Belmont, Sergey Svishchev
+/***************************************************************************
+
+    agat.c
+	Skeleton driver for Agat series of Soviet Apple II non-clones
+
+	These are similar to Apple II (same bus architecture, keyboard and
+	floppy interface), but video controller is completely different.
+
+	To do:
+	- native keyboards (at least two variants)
+	- 840K floppy controller (MFM encoding, but track layout is unique)
+	- agat7: 64K and 128K onboard memory configurations
+	- agat9
+	- 3rd party slot devices
+
+************************************************************************/
+
+#include "emu.h"
+#include "includes/apple2.h"
```

Comment:
It doesn't seem clear why this dependency should be necessary when the driver has been split and agat_state doesn't inherit from the apple2 driver class.